v2v-helper/migrate/migrate.go

```diff
@@ -229,8 +229,15 @@ func (migobj *Migrate) LiveReplicateDisks(ctx context.Context, vminfo vm.VMInfo)
 		}
 	}
 
+	// clean up snapshots
+	utils.PrintLog("Cleaning up snapshots before copy")
+	err := vmops.CleanUpSnapshots(false)
+	if err != nil {
+		return vminfo, fmt.Errorf("failed to clean up snapshots: %s", err)
```

Comment:
Can we make the error like, this:

```suggestion
		return vminfo, fmt.Errorf("failed to clean up snapshots: %s, Please delete manually before starting again,", err)
```

something like this: